### ISSUE TITLE
Add option to set caption (suggestion)

### ISSUE THREAD

**Issue by CNK001** [imgix/luminous#484]
It would be great if the gallery supported the caption option when zoomed in, for example from alt or title.

**Comment by frederickfogerty** [comment]
Hey there @CNK001, thanks for the idea! We've added this to our internal tracker and will post any updates here when we get around to this!

**Comment by ianhobbs** [comment]
Yes a deal breaker for me. But I see in the gallery mode caption is optioned?
